src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -2094,8 +2094,12 @@ struct ONNXElementwiseBinaryOpLowering
               shapeHelper, create, op, outputMemRefType,
               collapsedInnermostLoops, estimatedSimdLoopTripCount);
       if (uVL > 0) {
-        onnxToKrnlSimdReport(op, /*successful*/ true, uVL,
-            estimatedSimdLoopTripCount, "binary with manageable broadcast");
+        if (collapsedInnermostLoops == outputRank)
```

Comment:
@AlexandreEichenberger fyi, clang 14.0.3 on Mac warns:
```
Elementwise.cpp:2097:37: warning: comparison of integers of different signs: 'int64_t' (aka 'long long') and 'uint64_t' (aka 'unsigned long long') [-Wsign-compare]
        if (collapsedInnermostLoops == outputRank)
```

Reply:
@sorenlassen thanks, saw it too and fixed it in a subsequent PR. Thanks for the info